.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -254,20 +254,26 @@ class common_job_properties {
     return mapToArgString(joinedArgs)
   }
 
+  private static def buildPerfKit(def context) {
+    context.steps {
+      // Clean up environment.
+      shell('rm -rf PerfKitBenchmarker')
+      // Clone appropriate perfkit branch
+      shell('git clone https://github.com/GoogleCloudPlatform/PerfKitBenchmarker.git')
+      // Install Perfkit benchmark requirements.
+      shell('pip install --user -r PerfKitBenchmarker/requirements.txt')
+      // Install job requirements for Python SDK.
+      shell('pip install --user -e sdks/python/[gcp,test]')
+    }
+  }
+
   // Adds the standard performance test job steps.
   static def buildPerformanceTest(def context, def argMap) {
     def pkbArgs = genPerformanceArgs(argMap)
+    buildPerfKit(context)
     context.steps {
-        // Clean up environment.
-        shell('rm -rf PerfKitBenchmarker')
-        // Clone appropriate perfkit branch
-        shell('git clone https://github.com/GoogleCloudPlatform/PerfKitBenchmarker.git')
-        // Install Perfkit benchmark requirements.
-        shell('pip install --user -r PerfKitBenchmarker/requirements.txt')
-        // Install job requirements for Python SDK.
-        shell('pip install --user -e sdks/python/[gcp,test]')
-        // Launch performance test.
-        shell("python PerfKitBenchmarker/pkb.py $pkbArgs")
+      // Launch performance test.
+      shell("python PerfKitBenchmarker/pkb.py $pkbArgs")
```

Comment:
I don't know that you can have multiple `steps` sections, can you? I think this is one of the things with this DSL.

Reply:
This script used to work few weeks back. Did something change ?

Reply:
This PR seems to change it. The other failure is unrelated, but implies you are building not against the merge commit, but against the head of the PR. That seems like a bug itself in our Jenkins triggering, but you might just try rebasing.